scipy/signal/signaltools.py

```diff
@@ -488,7 +488,7 @@ def _reverse_and_conj(x):
     Reverse array `x` in all dimensions and perform the complex conjugate
     """
     reverse = [slice(None, None, -1)] * x.ndim
```

Comment:
Could just change this one to a tuple literal rather than a list

Reply:
OK, done